src/plugins/pendo/components/OptInModal/OptInModalBody.tsx

```diff
@@ -51,13 +58,8 @@ export const OptInModalBody: React.FC<OptInModalProps> = ({ onContinue }) => {
     onContinue()
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":57,"endLine":58,"prLink":"https://app.graphite.dev/github/pr/shapeshift/web/2573"}} -->
<i class='graphite__hidden'>[Re: lines 57 to 58]<br/></i>

Note, ideally, we'd be able to consolidate this in the `useEffect` but `consent` isn't reactive i.e we don't get a new reference on consent change, so this is fine<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/shapeshift/web/2573?utm_source=unchanged-line-comment">Graphite</a>.</span>